### PR TITLE
feature: ArraySingleMultiLine - Introduction

### PR DESCRIPTION
**Pull request by SpacePossum**
closes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/6225

### CODE REVIEW

File: src/Fixer/ArrayNotation/ArraySingleMultiLineFixer.php [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    /**
+     * {@inheritdoc}
+     */
+    protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
+    {
+        $assertIntMoreThanZero = [static function (int $value): bool { return $value > 0; }];
+
+        return new FixerConfigurationResolver([
+            (new FixerOptionBuilder('element_count', 'Threshold: # of elements an array must have be be written multiline.'))
+                ->setDefault(25)
```
Comment (kubawerlos) [inline src/Fixer/ArrayNotation/ArraySingleMultiLineFixer.php:98]: Can we not have these defaults, so anyone who wants to use the fixer must set the numbers by himself?

**Comment by SpacePossum** [comment]
maybe I pick this up later, for now I have no time for this any more.

**Comment by adjenks** [comment]
Sad this is gone, it's just what I was looking for. I just pasted it into my project for now. It seems to work fine.
Will you pick it up or should someone fork it and take over?

**Comment by SpacePossum** [comment]
I've no plans picking this one up anytime soon sadly. So if anyone else wants to pick it up would be great, :+1:  might be wise to discuss the expected behavior with maintainers first before putting time in it.

**Pull request closed without merging**